app/src/main/java/fr/free/nrw/commons/settings/Prefs.java

```diff
@@ -7,8 +7,8 @@
     public static final String DEFAULT_LICENSE = "defaultLicense";
 
     public static class Licenses {
-        public static final String CC_BY_SA_3 = "CC BY-SA";
-        public static final String CC_BY_3 = "CC BY";
+        public static final String CC_BY_SA_3 = "CC BY-SA 3.0";
+        public static final String CC_BY_3 = "CC BY 3.0";
```

Comment:
I think the value of those strings should be changed to be more explicit - it was likely just "CC BY" in the legacy app because 4.0 wasn't out yet, but now that it is, it should be renamed to "CC BY 3.0" etc. Should probably be in a different PR though, heh.